docs/integrating-into-frameworks.md

```diff
@@ -14,11 +14,10 @@ frameworks.
 
 ## Examples
```

Comment:
Question: do we want to maintain the list ourselves, or we are open to pull request since you listed the criteria here?

Reply:
To Yiran's point, would it make sense to re-word or augment "More coming soon!" to make it clear we're open to pull requests if someone thinks they have a good example repository?  "More coming soon!" might make it sound like we're planning to make them ourselves.